src/components/Vizmapper/VisualPropertyRender/Color.tsx

```diff
@@ -19,7 +19,171 @@ export function ColorPicker(props: {
+    '#717171', '#00a7d5', '#1200d8', '#b500d7', '#e800a6', '#e70000', '#dba000', '#d1d300', '#94d500', '#00d800', '#00d86a', '#00d8d3',
+    '#717171', '#00a8a4', '#0b00a8', '#b400a7', '#b400a7', '#b40000', '#ac6e00', '#a2a400', '#a2a400', '#00a800', '#00a86f', '#00a8a4',
+    '#3a3a3a', '#007472', '#050075', '#7e0074', '#7d0000', '#7d0000', '#717200', '#717200', '#717200', '#1c7400', '#007472', '#007472',
+    '#000000', '#003c3b', '#01003c', '#41003c', '#41003c', '#3a3a00', '#808900', '#3a3a00', '#3a3a00', '#003c3b', '#003c3b', '#3a3a3a',
+  ]
+
+  return (
+    <Box>
+      <CompactPicker
+        colors={customColors}
+        color={localColorValue}
+        onChange={(color: any) => {
+          setLocalColorValue(color.hex)
+          debouncedValueChange(color.hex)
+        }}
+      />
+    </Box>
+  )
+}
+
+export function ColorPickerViridis(props: {
+  currentValue: ColorType | null
+  onValueChange: (color: ColorType) => void
+}): React.ReactElement {
+  const { onValueChange, currentValue } = props
+  const debouncedValueChange = debounce(onValueChange, 200)
+
+  // use local state to appear instantaneous in the color picker,
+  // but the actual visual style model updates are debounced
+  const [localColorValue, setLocalColorValue] = React.useState<ColorType>(
+    currentValue ?? `#ffffff`,
+  )
+
+  const customColors = [
+    ['#ffea00', '#98e21e', '#00ce71', '#00af91', '#008f9a', '#2f689b', '#55378d', '#560061'],
+    ['#fcfec2', '#ffbf86', '#ff7762', '#ec3379', '#b31b8e', '#75008e', '#2d0c60', '#000001'],
+    ['#effb00', '#ffc100', '#ff8b46', '#f25872', '#d41995', '#a600b3', '#6b00b2', '#120096'],
+    ['#fbffa6', '#ffc600', '#ff7e00', '#ed3c46', '#ba106f', '#7c007c', '#350062', '#000001'],
+  ];
+
+  return (
+    <Box>
+      <SwatchesPicker
+        colors={customColors}
+        color={localColorValue}
+        onChange={(color: any) => {
+          setLocalColorValue(color.hex)
+          debouncedValueChange(color.hex)
+        }}
+      />
+    </Box>
+  )
+}
+
+export function ColorPickerSequential(props: {
+  currentValue: ColorType | null
+  onValueChange: (color: ColorType) => void
+}): React.ReactElement {
+  const { onValueChange, currentValue } = props
+  const debouncedValueChange = debounce(onValueChange, 200)
+
+  // use local state to appear instantaneous in the color picker,
+  // but the actual visual style model updates are debounced
+  const [localColorValue, setLocalColorValue] = React.useState<ColorType>(
+    currentValue ?? `#ffffff`,
+  )
+
+  const customColors = [
+    ['#f7fdf6', '#e5f7e3', '#c5eec5', '#9ae1a1', '#60cf7c', '#00b962', '#009a49', '#006836'],
+    ['#f7fdf1', '#e0f6de', '#caefca', '#a2e4bb', '#69d6cb', '#0bbfdb', '#009ac9', '#0065ac'],
+    ['#fff8fc', '#f1e5f3', '#d6d7ea', '#aac6e1', '#aac6e1', '#009eca', '#009096', '#00735a'],
+    ['#fff8f4', '#ffe2e1', '#ffc8c6', '#ffa2bd', '#ff62ac', '#f700a4', '#ca008c', '#940085'],
```

Comment:
These should be defined in the colors model impl so that they can be reused in other places in the app that needs to deal with colors.